build/configs/imxrt1020-evk/nxp_demo/defconfig

```diff
@@ -300,7 +302,9 @@ CONFIG_IMXRT_AUTOMOUNT_USERFS=y
 CONFIG_IMXRT_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p1"
 CONFIG_IMXRT_AUTOMOUNT_USERFS_MOUNTPOINT="/mnt"
 # CONFIG_IMXRT_AUTOMOUNT_SSSRW is not set
-# CONFIG_IMXRT_AUTOMOUNT_ROMFS is not set
+CONFIG_IMXRT_AUTOMOUNT_ROMFS=y
+CONFIG_IMXRT_AUTOMOUNT_ROMFS_DEVNAME="/dev/mtdblock9"
+CONFIG_IMXRT_AUTOMOUNT_ROMFS_MOUNTPOINT="/rom"
```

Comment:
If ROMFS is required, you need to change the CONFIG_IMXRT_FLASH_PART_xxxxx configs to add a new partition for romfs.